asciidoc/getting_started/playwright/plugin-information.adoc

```diff
@@ -0,0 +1,59 @@
+= @ensono-stacks/playwright
+:sidebar_label: @ensono-stacks/playwright
+:description: Using Playwright with Ensono Stacks
+:keywords: Nx, monorepo, stacks, ensono, playwright, generator, executor
+
+Using the `@ensono-stacks/playwright` plugin can help you get started with testing using Playwright. With its comprehensive feature set, this plugin accelerates your setup process and provides examples to get you started.
+
+Additionally, Playwright tests can be added to your deployment pipelines through various deployment generators.
+
+[NOTE]
+====
+Visit the link:../../testing/testing_in_nx/frontend_testing_in_nx.adoc[Frontend Testing in an Nx monorepo] and link:../../testing/testing_in_nx/playwright_nx.adoc[Testing with Playwright] user guides for more information!
+====
+
+== Setting up @ensono-stacks/playwright
+
+Install the `@ensono-stacks/playwright` plugin with the following command:
+
+== npm
+
+[source]
+----
+npm install --save-dev @ensono-stacks/playwright@latest
+----
+
+== yarn
+
+[source]
+----
+yarn add --dev @ensono-stacks/playwright@latest
+----
```

Comment:
For the npm and yarn code blocks i've just done the following. Don't mind if we use this style, but good to be consistent.
```
----
npm install --save-dev @ensono-stacks/playwright@latest
----

or

----
yarn add --dev @ensono-stacks/playwright@latest
----
```

Reply:
updated this